backend/src/graphql/resolver/CommunityResolver.ts

```diff
@@ -40,13 +41,41 @@ export class CommunityResolver {
     return dbCommunities.map((dbCom: DbCommunity) => new Community(dbCom))
   }
 
-  @Authorized([RIGHTS.COMMUNITIES])
+  @Authorized([RIGHTS.COMMUNITY_BY_UUID])
   @Query(() => Community)
-  async community(@Arg('communityUuid') communityUuid: string): Promise<Community> {
-    const community = await getCommunity(communityUuid)
-    if (!community) {
+  async getCommunityByUuid(@Arg('communityUuid') communityUuid: string): Promise<Community> {
+    const com: DbCommunity | null = await getCommunityByUuid(communityUuid)
```

Comment:
I like to use a shorter naming schema for the future and the refactoring. 
It's graphql not rest so I don't see the need for express the information more than once.
It is a query not a mutation so no need for get. All queries are for getting something.
You express the possible arguments already in the schema, no need to repeat that with ByUuid.

Reply:
Ok i agree for the graphql naming, but not for the typescript functions, rigth?

Reply:
you know we use [typed-graphql](https://typegraphql.com/docs/getting-started.html)
The graphql function names will be generated from the resolver function names.